composites/Plugin/DashboardWidget/components/WordpressFeed.js

```diff
@@ -59,21 +60,18 @@ const A11yNotice = styled.span`
 `;
 
 const WordpressFeedListItemContainer = styled.li`
-	padding: 0 8px 8px;
+	margin: 8px 0;
 	overflow: hidden;
 
 	a {
 		padding-bottom: 4px;
```

Comment:
wouldn't be better to put this bottom padding in the WordpressFeedLink component?